src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/IVSTypeScriptLanguageDebugInfoService.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api
+{
+    internal interface IVSTypeScriptLanguageDebugInfoService
```

Comment:
The name should be updated to distinguish implementation interfaces (implementation lives in partner code) from accessor interfaces (implementation lives in roslyn internal code).

```suggestion
    internal interface IVSTypeScriptLanguageDebugInfoServiceImplementation
```

Reply:
Aren't these already distinguished by the `IVSTypeScript` prefix?